<launch>
    <node name="scout_base_node" pkg="scout_base" type="scout_base_node" output="screen">
        <param name="port_name" type="string" value="can0" />

        <param name="odom_frame" type="string" value="odom" />
        <param name="base_frame" type="string" value="base_link" />

        <param name="simulated_robot" type="bool" value="false" />
    </node>
</launch>
